cmd/kubeadm/app/util/pkiutil/pki_helpers.go

```diff
@@ -201,7 +201,7 @@ func TryLoadKeyFromDisk(pkiPath, name string) (*rsa.PrivateKey, error) {
 	case *rsa.PrivateKey:
 		key = k
 	default:
-		return nil, errors.Wrapf(err, "the private key file %s isn't in RSA format", privateKeyPath)
+		return nil, errors.Errorf("the private key file %s isn't in RSA format", privateKeyPath)
```

Comment:
If you try to wrap a `nil` error, this will result in `nil` instead of a new error

Reply:
Good catch, fixed!